### PR TITLE
refactor: apply schema to notice apis & update to use notice store and notice detail store

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [ ] Not that difficult
- [ ] Approved feature branch merge to master

### Description
Please review in order by commits

### CODE REVIEW

**Review by yuda110 — APPROVED** [review]
LGTM!!!

File: apps/web/src/schema/board/board/api-verbs/list.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import type { ListResponse } from '@/schema/_common/api-verbs/list';
+import type { BoardModel } from '@/schema/board/board/model';
+
+export interface BoardListParameters {
+    board_id?: string;
+    name?: string;
+}
+
+export type BoardListResponse = ListResponse<BoardModel>;
```
Comment (sulmoJ) [inline apps/web/src/schema/board/board/api-verbs/list.ts:9]: 😆

File: apps/web/src/services/info/pages/admin/AdminNoticeDetailPage.vue [hunk 2]
```diff
@@ -66,11 +66,9 @@ const handleDeleteNoticeConfirm = async () => {
     }
 };
 
-onBeforeMount(() => {
-    noticeDetailStore.getNoticePost(props.postId);
-});
-onUnmounted(() => {
-    noticeDetailStore.reset();
+onBeforeMount(async () => {
+    noticeDetailStore.reset(); // do not reset on unmounted for the case of moving to update page
```
Comment (sulmoJ) [inline apps/web/src/services/info/pages/admin/AdminNoticeDetailPage.vue:70]: Ok 👍

**Review by sulmoJ — APPROVED** [review]
Clear~

**Pull request merged**